lib/cli/interactive-setup/console-dev-mode.js

```diff
@@ -0,0 +1,337 @@
+'use strict';
+
+const { writeText, style } = require('@serverless/utils/log');
+const WebSocket = require('ws');
+const chalk = require('chalk');
+const resolveAuthMode = require('@serverless/utils/auth/resolve-mode');
+const streamBuffers = require('stream-buffers');
+const apiRequest = require('@serverless/utils/api-request');
+
+const stage = process.env.SERVERLESS_PLATFORM_STAGE || 'prod';
+const socketUrl =
+  stage === 'prod'
+    ? 'wss://ehst4ktjyi.execute-api.us-east-1.amazonaws.com/prod'
+    : 'wss://4c684ym73e.execute-api.us-east-1.amazonaws.com/dev';
```

Comment:
Should I move this into `@serverless/utils`? I wasn't sure if it made more sense there or just in here 🤔

Reply:
Yes @medikoo I think either way we need to fix the urls. I am planning on updating these urls today 👍